packages/twenty-server/src/metadata/data-source/data-source.entity.ts

```diff
@@ -10,13 +10,16 @@ import {
 
 import { ObjectMetadataEntity } from 'src/metadata/object-metadata/object-metadata.entity';
 
-type DataSourceType = DataSourceOptions['type'];
+export type DataSourceType = DataSourceOptions['type'];
 
 @Entity('dataSource')
 export class DataSourceEntity {
   @PrimaryGeneratedColumn('uuid')
   id: string;
 
+  @Column({ nullable: false })
```

Comment:
Why is it non-nullable now? I actually believe we never use that field right?

Reply:
Hmm good catch, I change it at first so we can query datasources by their name, I'll revert this !